as-vending/main.go

```diff
@@ -57,46 +77,48 @@ func main() {
 	inferenceStopChannel := make(chan int)
 
 	// Set default values for vending state
-	vendingState.CVWorkflowStarted = false
-	vendingState.MaintenanceMode = false
-	vendingState.CurrentUserData = functions.OutputData{}
-	vendingState.DoorClosed = true
+	app.vendingState.CVWorkflowStarted = false
+	app.vendingState.MaintenanceMode = false
+	app.vendingState.CurrentUserData = functions.OutputData{}
+	app.vendingState.DoorClosed = true
 	// global stop channel for threads
-	vendingState.ThreadStopChannel = stopChannel
+	app.vendingState.ThreadStopChannel = stopChannel
 	// open event thread
-	vendingState.DoorOpenedDuringCVWorkflow = false
-	vendingState.DoorOpenWaitThreadStopChannel = doorOpenStopChannel
+	app.vendingState.DoorOpenedDuringCVWorkflow = false
+	app.vendingState.DoorOpenWaitThreadStopChannel = doorOpenStopChannel
 	// close event thread
-	vendingState.DoorClosedDuringCVWorkflow = false
-	vendingState.DoorCloseWaitThreadStopChannel = doorCloseStopChannel
+	app.vendingState.DoorClosedDuringCVWorkflow = false
+	app.vendingState.DoorCloseWaitThreadStopChannel = doorCloseStopChannel
 	// inference thread
-	vendingState.InferenceDataReceived = false
-	vendingState.InferenceWaitThreadStopChannel = inferenceStopChannel
+	app.vendingState.InferenceDataReceived = false
+	app.vendingState.InferenceWaitThreadStopChannel = inferenceStopChannel
```

Comment:
80-94 would be a good candidate for a new vendingStateInitialize function to make the code a bit easier to read.

Reply:
moving this to an issue